h2/src/main/org/h2/command/Parser.java

```diff
@@ -7883,6 +7886,11 @@ private Prepared parseSet() {
             readIfEqualOrTo();
             read();
             return new NoOperation(session);
+        } else if (readIf("JOIN_COLLAPSE_LIMIT")) {
+            // for PostgreSQL compatibility
+            readIfEqualOrTo();
+            read();
+            return new NoOperation(session);
```

Comment:
This code too, we already have too many non-standard stuff that is incorrectly accepted in all compatibility modes.